pussy/Program.cs

```diff
@@ -49,5 +65,87 @@ static void Main()
         }
 
     }
-    
+    public sealed class Wallpaper
+    {
+        Wallpaper() { }
+
+        const int SPI_SETDESKWALLPAPER = 20;
+        const int SPIF_UPDATEINIFILE = 0x01;
+        const int SPIF_SENDWININICHANGE = 0x02;
+
+        [DllImport("user32.dll", CharSet = CharSet.Auto)]
+        static extern int SystemParametersInfo(int uAction, int uParam, string lpvParam, int fuWinIni);
+
+        public enum Style : int
+        {
+            Fill,
+            Fit,
+            Span,
+            Stretch,
+            Tile,
+            Center
+        }
+
+        public static void Set(String wpaper, Style style)
+        {
+            
+            System.Drawing.Image img = System.Drawing.Image.FromFile(Path.GetFullPath(wpaper));
+            string tempPath = Path.Combine(Path.GetTempPath(), "wallpaper.bmp");
+            img.Save(tempPath, System.Drawing.Imaging.ImageFormat.Bmp);
+
+            RegistryKey key = Registry.CurrentUser.OpenSubKey(@"Control Panel\Desktop", true);
+            if (style == Style.Fill)
+            {
+                key.SetValue(@"WallpaperStyle", 10.ToString());
```

Comment:
what the actual fuck '10.ToString()' is

Reply:
im either too clever or too dumb

Reply:
@Cherlyonok please help me comprehend it

Reply:
And also do we even need the '@' symbols before all these string literals? I doubt it.\
*AFAIK they are used when the string literal contains specific characters (e.g. backslashes) which are to be escaped.*

Reply:
> 
well, obviously it can be replaced with "10" in this context. ToString() method was used to play with different numeric values as Number objects. I'll "fix" it in the next commit

Reply:
> And also do we even need the `@` symbols before all these string literals? I doubt it. _AFAIK they are used when the string literal contains specific characters (e.g. backslashes) which are to be escaped._

yeah i fucked up, copied @"" format from the RegistryKey key. The author of the desktop work tutorial did the same and it somehow slipped my eye. Now fixed